test/unit/query/builder.js

```diff
@@ -2,21 +2,31 @@
 /*eslint no-var:0, indent:0, max-len:0 */
 'use strict';
 
-var MySQL_Client = require('../../../lib/dialects/mysql')
-var PG_Client = require('../../../lib/dialects/postgres')
-var Oracle_Client = require('../../../lib/dialects/oracle')
-var Oracledb_Client = require('../../../lib/dialects/oracledb')
-var SQLite3_Client = require('../../../lib/dialects/sqlite3')
-var MSSQL_Client = require('../../../lib/dialects/mssql')
+var _typeof2 = require('babel-runtime/helpers/typeof');
+
+var _typeof3 = _interopRequireDefault(_typeof2);
+
+var _keys = require('babel-runtime/core-js/object/keys');
+
+var _keys2 = _interopRequireDefault(_keys);
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+var MySQL_Client = require('../../../lib/dialects/mysql');
+var PG_Client = require('../../../lib/dialects/postgres');
+var Oracle_Client = require('../../../lib/dialects/oracle');
+var Oracledb_Client = require('../../../lib/dialects/oracledb');
+var SQLite3_Client = require('../../../lib/dialects/sqlite3');
+var MSSQL_Client = require('../../../lib/dialects/mssql');
 
 var clients = {
   mysql: new MySQL_Client({}),
   postgres: new PG_Client({}),
   oracle: new Oracle_Client({}),
   oracledb: new Oracledb_Client({}),
   sqlite3: new SQLite3_Client({}),
-  mssql: new MSSQL_Client({}),
-}
+  mssql: new MSSQL_Client({})
```

Comment:
You shouldn't do styling changes in the same pull request that is doing other type of changes. This PR is huge enough without these, It makes reviewing harder since I have to check for every line if it is functional change or not. Specially important with this one, since I've skimmed through this already multiple times.